Utils/github_workflow_scripts/get_xsoar_supported_release_notes.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python3
+"""
+This script is used in the Review Release Notes GitHub workflow.
+It accepts a list of changed files (release notes)
+and returns the release notes of XSOAR-supported Packs.
+"""
+
+from typing import Dict, Any, List
+import json
+from pathlib import Path
+import argparse
+import sys
+import traceback
+import click
+
+
+def is_pack_xsoar_supported(pack_name: str) -> bool:
+    """
+    Checks whether the supplied Pack is from a XSOAR-supported
+
+    Args:
+        - `pack_name` (``str``): The name of the Pack
+
+    Returns:
+        `bool` representing whether Pack is XSOAR-supported or not
+    """
+
+    pack_metadata = Path("Packs", pack_name, "pack_metadata.json")
+
+    if pack_metadata.exists():
+        with pack_metadata.open(encoding="UTF-8") as pm:
+            metadata: Dict[str, Any] = json.load(pm)
+            if metadata.get("support") == "xsoar":
+                return True
+
+    return False
+
+
+def convert_files_to_paths(files: List[str]) -> List[Path]:
+    """
+    Converts a list of relative file paths to a list of `Path` objects for
+    easier navigation of filesystem.
+
+    Args:
+        - `files` (``List[str]``): The relative paths to the file.
+
+    Returns:
+        `List[Path]` representing a list of filesystem files.
+    """
+
+    return list(map(lambda f: Path(f), files))
+
+
+def format_output(rns_to_review: List[str], delimiter: str = ",") -> str:
+    """
+    Convert the list of release notes to review into a comma-separated string
+    in preparation for output.
+
+    Args:
+        - `rns_to_review` (``List[str]``): List of release note paths.
+        - `delimiter` (`str`): The delimiter to separate the release note paths
+
+    Returns"
+        `str` of joined release notes to review
+    """
+
+    return delimiter.join(rns_to_review)
```

Comment:
I see no need for a whole function declaration just for this line

Reply:
I think it actually makes it easier to test out the expected script output as specified in [UT](https://github.com/demisto/content/blob/CIAC-4970-run-release-notes-check-only-if-xsoar-supported-pa/Utils/github_workflow_scripts/github_workflow_scripts_tests/test_get_xsoar_supported_release_notes.py#L80).